feat: change template and examples [MONET-1715] (#6191)
contentful/apps: examples/function-templates/javascript/INSTRUCTIONS.md

It as simple using the CLI command `npm run upload-ci`. This will perform two actions: upload the code, linking it to the app, and then finally activating the code ready for usage in both 

### Assigning an app to a field

> Functions are meant to help with resolving field data, meaning the app has to be assigned to a field location. You can read more about different app locations [here](https://www.contentful.com/developers/docs/extensibility/app-framework/locations/). 

Reviewer: i don't understand these DF readme instruction changes... you HAVE to be specific about specific function context instructions at some point and renaming all of this here doesn't even make sense to me... ?

Reviewer: ```suggestion
Functions that support resolving field data on the GraphQL API have to be assigned to a field location. You can read more about different app locations [here](https://www.contentful.com/developers/docs/extensibility/app-framework/locations/). 
```

Reviewer: Perhaps this suggestion helps clarify my concerns/how we should frame this to developers?

Author: perfect, thanks!

> The field is required to be one of the supported types for function: Short text (`Symbol`) or JSON fields (`Object`).

Reviewer: ```suggestion
Supported field types for functions are: Short text (`Symbol`) or JSON fields (`Object`).
```
